### PR TITLE
feat: add python3 code for hot-link and PayPerView protection

### PR DESCRIPTION
**Pull request by slim0**
Hi,

I hope you are doing well.
Since I'm using Nimble with WMS Panel on a project running on python3, I had to modify python/wmsauth.py and create python/ppv_media_signature.py based on your php code. 
I also added functions, documentation, and 'signed_stream' parameter to add it into the signature.

Best Regards.

Simon

### CODE REVIEW

File: python/wmsauth.py [hunk 1]
```diff
@@ -1,24 +1,31 @@
...
-m.update(ip + password + today + validminutes)
-
-base64hash = base64.b64encode(m.digest())
-
-urlsignature = "server_time=" + today + "&hash_value=" + base64hash + "&validminutes=" + validminutes
-
-base64urlsignature = base64.b64encode(urlsignature)
-
-signedurlwithvalidinterval = initial_url + "?wmsAuthSign=" + base64urlsignature
+def get_media_url_with_signature(ip, password, validminutes, initial_url, signed_stream, strm_len):
```
Comment (alexpokotilo) [inline python/wmsauth.py]: set default values for signed_stream="" and stream_len=0

File: python/ppv_media_signature.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+from hashlib import md5
+from time import gmtime, strftime
+
+
+def ppv_url_with_signature(ip, user_id, password, validminutes, initial_url, signed_stream, strm_len):
+    """
+    Get URL with signature when using PayPerView method with Nimble
+    :param ip: Viewer IP address
+    :param user_id: User ID used in back-end web server
+    :param password: hot-link protection password wonfigured in WMSPanel Auth Rule
```
Comment (alexpokotilo) [inline python/ppv_media_signature.py]: wonfigred->configured

File: python/ppv_media_signature.py [hunk 3]
```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python3
+# coding: utf8
+
+import base64
+from hashlib import md5
+from time import gmtime, strftime
+
+
+def ppv_url_with_signature(ip, user_id, password, validminutes, initial_url, signed_stream, strm_len):
```
Comment (alexpokotilo) [inline python/ppv_media_signature.py]: set default values for signed_stream="" and stream_len=0

**Comment by alexpokotilo** [comment]
Please confirm your contribution is under MIT license

**Comment by slim0** [comment]
> Please confirm your contribution is under MIT license

Yes it is !

**Comment by alexpokotilo** [comment]
thanks! Please check code review comments above

**Review by slim0 — COMMENTED** [review]
Code review asked by Alex Pokotilo.

**Review by alexpokotilo — APPROVED** [review]
it's fine now! Thanks a lot!

**Comment by WMSPanel** [comment]
Thank you very much for your contribution!

**Pull request merged**